src/Microsoft.NetCore.Analyzers/Core/Tasks/DoNotCreateTaskCompletionSourceWithWrongArguments.Fixer.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.NetCore.Analyzers.Tasks
+{
+    /// <summary>
+    /// RS1027: Do not create TaskCompletionSource with wrong arguments
+    /// </summary>
+    public sealed class DoNotCreateTaskCompletionSourceWithWrongArgumentsFixer : CodeFixProvider
```

Comment:
Please delete this file unless you are planning to implement this code fixer in this PR.

Reply:
I plan to add a fixer. Was going to do it in another PR but I understand your point. Will decide shortly to delete or add fixer with this PR.